`MPIBenchmark` build issue

**Describe the bug**

Parallel benchmark build fails:
```
[ 90%] Building CXX object CMakeFiles/8_benchmark_parallel.dir/examples/8_benchmark_parallel.cpp.o
In file included from /home/pgrete/src/openPMD-api/examples/8_benchmark_parallel.cpp:1:
/home/pgrete/src/openPMD-api/include/openPMD/benchmark/mpi/MPIBenchmark.hpp: In member function ‘void openPMD::MPIBenchmark<DatasetFillerProvider>::resetConfigurations()’:
/home/pgrete/src/openPMD-api/include/openPMD/benchmark/mpi/MPIBenchmark.hpp:303:11: error: ‘class openPMD::MPIBenchmark<DatasetFillerProvider>’ has no member named ‘m_compressions’ [-Wtemplate-body]
  303 |     this->m_compressions.clear();
      |           ^~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/8_benchmark_parallel.dir/build.make:79: CMakeFiles/8_benchmark_parallel.dir/examples/8_benchmark_parallel.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1847: CMakeFiles/8_benchmark_parallel.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

**To Reproduce**

```bash
$ cmake -Bbuild-dev -DCMAKE_INSTALL_PREFIX=install-dev
-- The C compiler identification is GNU 15.1.1
-- The CXX compiler identification is GNU 15.1.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- openPMD-api superbuild: ON
-- Found MPI_CXX: /usr/lib/libmpi.so (found version "3.1")
-- Found MPI: TRUE (found version "3.1") found components: CXX
-- Downloading nlohmann_json ...
-- nlohmann_json source: https://github.com/nlohmann/json/archive/refs/tags/v3.11.3.tar.gz
-- Using the multi-header code from /home/pgrete/src/openPMD-api/build-dev/_deps/fetchednlohmann_json-src/include/
-- Downloading toml11 ...
-- toml11 source: https://github.com/ToruNiina/toml11/archive/refs/tags/v4.2.0.tar.gz
CMake Deprecation Warning at build-dev/_deps/fetchedtoml11-src/CMakeLists.txt:23 (cmake_policy):
  The OLD behavior for policy CMP0127 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.


-- Performing Test TOML11_COMPILER_SUPPORTS_WALL
-- Performing Test TOML11_COMPILER_SUPPORTS_WALL - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WEXTRA
-- Performing Test TOML11_COMPILER_SUPPORTS_WEXTRA - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WPEDANTIC
-- Performing Test TOML11_COMPILER_SUPPORTS_WPEDANTIC - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WERROR
-- Performing Test TOML11_COMPILER_SUPPORTS_WERROR - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WSIGN_CONVERSION
-- Performing Test TOML11_COMPILER_SUPPORTS_WSIGN_CONVERSION - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WCONVERSION
-- Performing Test TOML11_COMPILER_SUPPORTS_WCONVERSION - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WDUPLICATED_COND
-- Performing Test TOML11_COMPILER_SUPPORTS_WDUPLICATED_COND - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WDUPLICATED_BRANCHES
-- Performing Test TOML11_COMPILER_SUPPORTS_WDUPLICATED_BRANCHES - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WLOGICAL_OP
-- Performing Test TOML11_COMPILER_SUPPORTS_WLOGICAL_OP - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WDOUBLE_PROMOTION
-- Performing Test TOML11_COMPILER_SUPPORTS_WDOUBLE_PROMOTION - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WRANGE_LOOP_ANALYSIS
-- Performing Test TOML11_COMPILER_SUPPORTS_WRANGE_LOOP_ANALYSIS - Failed
-- Performing Test TOML11_COMPILER_SUPPORTS_WUNDEF
-- Performing Test TOML11_COMPILER_SUPPORTS_WUNDEF - Success
-- Performing Test TOML11_COMPILER_SUPPORTS_WSHADOW
-- Performing Test TOML11_COMPILER_SUPPORTS_WSHADOW - Success
-- Found HDF5: hdf5-shared (found version "1.14.6") found components: C
-- Found MPI_C: /usr/lib/libmpi.so (found version "3.1")
-- Found MPI: TRUE (found version "3.1")
-- Found ADIOS2: /usr/lib/cmake/adios2/adios2-config.cmake (found suitable version "2.10.2", minimum required is "2.9.0") found components: CXX MPI
-- Found Python: /usr/bin/python3.13 (found suitable version "3.13.3", minimum required is "3.7.0") found components: Interpreter Development.Module
-- Downloading pybind11 ...
-- pybind11 source: https://github.com/pybind/pybind11/archive/refs/tags/v2.13.6.tar.gz
CMake Deprecation Warning at build-dev/_deps/fetchedpybind11-src/CMakeLists.txt:13 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


-- pybind11 v2.13.6 
-- Performing Test HAS_FLTO
-- Performing Test HAS_FLTO - Success
-- Downloading Catch2 ...
-- Catch2 source: https://github.com/catchorg/Catch2/archive/refs/tags/v2.13.10.tar.gz
CMake Deprecation Warning at build-dev/_deps/fetchedcatch2-src/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


-- Python LTO/IPO: ON
-- Note: Skipping example and tool runs (missing openPMD-example-datasets)
-- Note: run
    /home/pgrete/src/openPMD-api/share/openPMD/download_samples.sh
to add example files to samples/git-sample/ directory!
-- Found mpi4py: TRUE

openPMD build configuration:
  library Version: 0.17.0
  openPMD Standard: 1.1.0
  C++ Compiler: GNU 15.1.1 
    /usr/bin/c++

  Install with RPATHs: ON
  Installation prefix: /home/pgrete/src/openPMD-api/install-dev
        bin: bin
        lib: lib
    include: include
      cmake: lib/cmake/openPMD
     python: lib/python3.13/site-packages

  Build Type: Release
  Library: shared
  CLI Tools: ON
  Examples: ON
  Testing: ON
  Invasive Tests: OFF
  Internal VERIFY: ON
  Build Options:
    MPI: ON
    HDF5: ON
    ADIOS2: ON
    PYTHON: ON
    CUDA_EXAMPLES: OFF

-- Configuring done (8.0s)
-- Generating done (0.1s)
-- Build files have been written to: /home/pgrete/src/openPMD-api/build-dev
[12:42:56][pgrete@haerke: ~/src/openPMD-api]
$ cmake --build /home/pgrete/src/openPMD-api/build-dev -j4
[  1%] Building CXX object CMakeFiles/CatchRunner.dir/test/CatchRunner.cpp.o
[  2%] Building CXX object CMakeFiles/CatchMain.dir/test/CatchMain.cpp.o
[  3%] Building CXX object CMakeFiles/openPMD.dir/src/ChunkInfo.cpp.o
[  3%] Building CXX object CMakeFiles/openPMD.dir/src/config.cpp.o
[  4%] Building CXX object CMakeFiles/openPMD.dir/src/Dataset.cpp.o
[  4%] Building CXX object CMakeFiles/openPMD.dir/src/Datatype.cpp.o
[  5%] Building CXX object CMakeFiles/openPMD.dir/src/Error.cpp.o
[  6%] Building CXX object CMakeFiles/openPMD.dir/src/Format.cpp.o
[  7%] Building CXX object CMakeFiles/openPMD.dir/src/Iteration.cpp.o
[  7%] Building CXX object CMakeFiles/openPMD.dir/src/IterationEncoding.cpp.o
[  8%] Building CXX object CMakeFiles/openPMD.dir/src/Mesh.cpp.o
[  9%] Building CXX object CMakeFiles/openPMD.dir/src/ParticlePatches.cpp.o
[ 10%] Linking CXX shared library lib/libCatchRunner.so
[ 10%] Linking CXX shared library lib/libCatchMain.so
[ 10%] Building CXX object CMakeFiles/openPMD.dir/src/ParticleSpecies.cpp.o
[ 10%] Built target CatchMain
[ 10%] Built target CatchRunner
[ 11%] Building CXX object CMakeFiles/openPMD.dir/src/Record.cpp.o
[ 12%] Building CXX object CMakeFiles/openPMD.dir/src/ReadIterations.cpp.o
[ 13%] Building CXX object CMakeFiles/openPMD.dir/src/RecordComponent.cpp.o
[ 13%] Building CXX object CMakeFiles/openPMD.dir/src/Series.cpp.o
[ 14%] Building CXX object CMakeFiles/openPMD.dir/src/UnitDimension.cpp.o
[ 15%] Building CXX object CMakeFiles/openPMD.dir/src/version.cpp.o
[ 15%] Building CXX object CMakeFiles/openPMD.dir/src/auxiliary/Date.cpp.o
[ 16%] Building CXX object CMakeFiles/openPMD.dir/src/auxiliary/Filesystem.cpp.o
[ 17%] Building CXX object CMakeFiles/openPMD.dir/src/auxiliary/JSON.cpp.o
[ 18%] Building CXX object CMakeFiles/openPMD.dir/src/auxiliary/JSONMatcher.cpp.o
[ 18%] Building CXX object CMakeFiles/openPMD.dir/src/auxiliary/Mpi.cpp.o
[ 19%] Building CXX object CMakeFiles/openPMD.dir/src/backend/Attributable.cpp.o
[ 20%] Building CXX object CMakeFiles/openPMD.dir/src/backend/BaseRecordComponent.cpp.o
[ 20%] Building CXX object CMakeFiles/openPMD.dir/src/backend/MeshRecordComponent.cpp.o
[ 21%] Building CXX object CMakeFiles/openPMD.dir/src/backend/PatchRecord.cpp.o
[ 22%] Building CXX object CMakeFiles/openPMD.dir/src/backend/PatchRecordComponent.cpp.o
[ 23%] Building CXX object CMakeFiles/openPMD.dir/src/backend/Writable.cpp.o
[ 23%] Building CXX object CMakeFiles/openPMD.dir/src/benchmark/mpi/OneDimensionalBlockSlicer.cpp.o
[ 24%] Building CXX object CMakeFiles/openPMD.dir/src/helper/list_series.cpp.o
[ 25%] Building CXX object CMakeFiles/openPMD.dir/src/snapshots/ContainerImpls.cpp.o
[ 26%] Building CXX object CMakeFiles/openPMD.dir/src/snapshots/ContainerTraits.cpp.o
[ 26%] Building CXX object CMakeFiles/openPMD.dir/src/snapshots/IteratorHelpers.cpp.o
[ 27%] Building CXX object CMakeFiles/openPMD.dir/src/snapshots/IteratorTraits.cpp.o
[ 28%] Building CXX object CMakeFiles/openPMD.dir/src/snapshots/RandomAccessIterator.cpp.o
[ 28%] Building CXX object CMakeFiles/openPMD.dir/src/snapshots/Snapshots.cpp.o
[ 29%] Building CXX object CMakeFiles/openPMD.dir/src/snapshots/StatefulIterator.cpp.o
[ 30%] Building CXX object CMakeFiles/openPMD.dir/src/IO/AbstractIOHandler.cpp.o
[ 31%] Building CXX object CMakeFiles/openPMD.dir/src/IO/AbstractIOHandlerImpl.cpp.o
[ 31%] Building CXX object CMakeFiles/openPMD.dir/src/IO/AbstractIOHandlerHelper.cpp.o
[ 32%] Building CXX object CMakeFiles/openPMD.dir/src/IO/Access.cpp.o
[ 33%] Building CXX object CMakeFiles/openPMD.dir/src/IO/DummyIOHandler.cpp.o
[ 33%] Building CXX object CMakeFiles/openPMD.dir/src/IO/IOTask.cpp.o
[ 34%] Building CXX object CMakeFiles/openPMD.dir/src/IO/FlushParams.cpp.o
[ 35%] Building CXX object CMakeFiles/openPMD.dir/src/IO/HDF5/HDF5IOHandler.cpp.o
[ 36%] Building CXX object CMakeFiles/openPMD.dir/src/IO/HDF5/ParallelHDF5IOHandler.cpp.o
[ 36%] Building CXX object CMakeFiles/openPMD.dir/src/IO/HDF5/HDF5Auxiliary.cpp.o
[ 37%] Building CXX object CMakeFiles/openPMD.dir/src/IO/JSON/JSONIOHandler.cpp.o
[ 38%] Building CXX object CMakeFiles/openPMD.dir/src/IO/JSON/JSONIOHandlerImpl.cpp.o
[ 38%] Building CXX object CMakeFiles/openPMD.dir/src/IO/JSON/JSONFilePosition.cpp.o
[ 39%] Building CXX object CMakeFiles/openPMD.dir/src/IO/ADIOS/ADIOS2IOHandler.cpp.o
[ 40%] Building CXX object CMakeFiles/openPMD.dir/src/IO/ADIOS/ADIOS2PreloadAttributes.cpp.o
[ 41%] Building CXX object CMakeFiles/openPMD.dir/src/IO/ADIOS/ADIOS2PreloadVariables.cpp.o
[ 41%] Building CXX object CMakeFiles/openPMD.dir/src/IO/ADIOS/ADIOS2File.cpp.o
[ 42%] Building CXX object CMakeFiles/openPMD.dir/src/IO/ADIOS/ADIOS2Auxiliary.cpp.o
[ 43%] Building CXX object CMakeFiles/openPMD.dir/src/IO/InvalidatableFile.cpp.o
[ 44%] Linking CXX shared library lib/libopenPMD.so
[ 44%] Built target openPMD
[ 46%] Building CXX object CMakeFiles/AuxiliaryTests.dir/test/AuxiliaryTest.cpp.o
[ 46%] Building CXX object CMakeFiles/CoreTests.dir/test/CoreTest.cpp.o
[ 46%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/openPMD.cpp.o
[ 46%] Building CXX object CMakeFiles/SerialIOTests.dir/test/SerialIOTest.cpp.o
[ 47%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/auxiliary.cpp.o
[ 48%] Linking CXX executable bin/AuxiliaryTests
[ 48%] Built target AuxiliaryTests
[ 48%] Building CXX object CMakeFiles/SerialIOTests.dir/test/Files_SerialIO/close_and_reopen_test.cpp.o
[ 49%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Access.cpp.o
[ 49%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Attributable.cpp.o
[ 50%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/BaseRecordComponent.cpp.o
[ 51%] Building CXX object CMakeFiles/CoreTests.dir/test/Files_Core/automatic_variable_encoding.cpp.o
[ 52%] Linking CXX executable bin/CoreTests
[ 52%] Built target CoreTests
[ 53%] Building CXX object CMakeFiles/SerialIOTests.dir/test/Files_SerialIO/filebased_write_test.cpp.o
[ 54%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/ChunkInfo.cpp.o
[ 55%] Building CXX object CMakeFiles/SerialIOTests.dir/test/Files_SerialIO/issue_1744_unique_ptrs_at_close_time.cpp.o
[ 56%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Dataset.cpp.o
[ 56%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Datatype.cpp.o
[ 57%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Error.cpp.o
[ 58%] Building CXX object CMakeFiles/ParallelIOTests.dir/test/ParallelIOTest.cpp.o
[ 59%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Helper.cpp.o
[ 60%] Building CXX object CMakeFiles/JSONTests.dir/test/JSONTest.cpp.o
[ 60%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Iteration.cpp.o
[ 61%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/IterationEncoding.cpp.o
[ 62%] Building CXX object CMakeFiles/ParallelIOTests.dir/test/Files_ParallelIO/read_variablebased_randomaccess.cpp.o
[ 63%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Mesh.cpp.o
[ 63%] Linking CXX executable bin/JSONTests
[ 63%] Built target JSONTests
[ 64%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/ParticlePatches.cpp.o
[ 65%] Linking CXX executable bin/SerialIOTests
[ 65%] Built target SerialIOTests
[ 65%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/ParticleSpecies.cpp.o
[ 65%] Building CXX object CMakeFiles/ParallelIOTests.dir/test/Files_ParallelIO/iterate_nonstreaming_series.cpp.o
[ 66%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/PatchRecord.cpp.o
[ 67%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/PatchRecordComponent.cpp.o
[ 67%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Record.cpp.o
[ 68%] Building CXX object CMakeFiles/ParallelIOTests.dir/test/Files_ParallelIO/bug_1655_bp5_writer_hangup.cpp.o
[ 69%] Building CXX object CMakeFiles/openpmd-ls.dir/src/cli/ls.cpp.o
[ 70%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/RecordComponent.cpp.o
[ 71%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/MeshRecordComponent.cpp.o
[ 72%] Linking CXX executable bin/ParallelIOTests
[ 73%] Linking CXX executable bin/openpmd-ls
[ 73%] Built target openpmd-ls
[ 73%] Building CXX object CMakeFiles/1_structure.dir/examples/1_structure.cpp.o
[ 73%] Built target ParallelIOTests
[ 74%] Building CXX object CMakeFiles/2_read_serial.dir/examples/2_read_serial.cpp.o
[ 75%] Linking CXX executable bin/1_structure
[ 75%] Built target 1_structure
[ 76%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/Series.cpp.o
[ 76%] Building CXX object CMakeFiles/openPMD.py.dir/src/binding/python/UnitDimension.cpp.o
[ 77%] Linking CXX executable bin/2_read_serial
[ 77%] Built target 2_read_serial
[ 77%] Building CXX object CMakeFiles/2a_read_thetaMode_serial.dir/examples/2a_read_thetaMode_serial.cpp.o
[ 78%] Building CXX object CMakeFiles/3_write_serial.dir/examples/3_write_serial.cpp.o
[ 79%] Building CXX object CMakeFiles/3a_write_thetaMode_serial.dir/examples/3a_write_thetaMode_serial.cpp.o
[ 80%] Linking CXX shared module lib/python3.13/site-packages/openpmd_api/openpmd_api_cxx.cpython-313-x86_64-linux-gnu.so
[ 80%] Linking CXX executable bin/3_write_serial
[ 80%] Built target 3_write_serial
[ 81%] Building CXX object CMakeFiles/3b_write_resizable_particles.dir/examples/3b_write_resizable_particles.cpp.o
[ 82%] Linking CXX executable bin/2a_read_thetaMode_serial
[ 82%] Built target 2a_read_thetaMode_serial
[ 83%] Building CXX object CMakeFiles/4_read_parallel.dir/examples/4_read_parallel.cpp.o
[ 84%] Linking CXX executable bin/3a_write_thetaMode_serial
[ 84%] Built target 3a_write_thetaMode_serial
[ 84%] Linking CXX executable bin/3b_write_resizable_particles
[ 84%] Built target 3b_write_resizable_particles
[ 84%] Building CXX object CMakeFiles/5_write_parallel.dir/examples/5_write_parallel.cpp.o
[ 85%] Linking CXX executable bin/4_read_parallel
[ 85%] Built target 4_read_parallel
[ 86%] Building CXX object CMakeFiles/6_dump_filebased_series.dir/examples/6_dump_filebased_series.cpp.o
[ 86%] Building CXX object CMakeFiles/7_extended_write_serial.dir/examples/7_extended_write_serial.cpp.o
[ 87%] Linking CXX executable bin/5_write_parallel
[ 87%] Built target 5_write_parallel
[ 88%] Building CXX object CMakeFiles/8_benchmark_parallel.dir/examples/8_benchmark_parallel.cpp.o
[ 89%] Linking CXX executable bin/6_dump_filebased_series
In file included from /home/pgrete/src/openPMD-api/examples/8_benchmark_parallel.cpp:1:
/home/pgrete/src/openPMD-api/include/openPMD/benchmark/mpi/MPIBenchmark.hpp: In member function ‘void openPMD::MPIBenchmark<DatasetFillerProvider>::resetConfigurations()’:
/home/pgrete/src/openPMD-api/include/openPMD/benchmark/mpi/MPIBenchmark.hpp:303:11: error: ‘class openPMD::MPIBenchmark<DatasetFillerProvider>’ has no member named ‘m_compressions’ [-Wtemplate-body]
  303 |     this->m_compressions.clear();
      |           ^~~~~~~~~~~~~~
[ 89%] Built target 6_dump_filebased_series
[ 89%] Building CXX object CMakeFiles/8a_benchmark_write_parallel.dir/examples/8a_benchmark_write_parallel.cpp.o
[ 90%] Linking CXX executable bin/7_extended_write_serial
[ 90%] Built target 7_extended_write_serial
[ 91%] Building CXX object CMakeFiles/8b_benchmark_read_parallel.dir/examples/8b_benchmark_read_parallel.cpp.o
make[2]: *** [CMakeFiles/8_benchmark_parallel.dir/build.make:79: CMakeFiles/8_benchmark_parallel.dir/examples/8_benchmark_parallel.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1847: CMakeFiles/8_benchmark_parallel.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 92%] Linking CXX executable bin/8a_benchmark_write_parallel
[ 92%] Built target 8a_benchmark_write_parallel
[ 92%] Built target openPMD.py
[ 92%] Linking CXX executable bin/8b_benchmark_read_parallel
[ 92%] Built target 8b_benchmark_read_parallel
make: *** [Makefile:146: all] Error 2


```


**Expected behavior**

Working binary.

**Software Environment**
 - version of openPMD-api: 0.16.1 and current `dev`
 - installed openPMD-api via: src
 - operating system: arch linux
 - machine: local workstation
 - name and version of Python implementation: [e.g. CPython 3.9]
 - version of HDF5: [e.g. 1.12.0]
 - version of ADIOS2: [e.g. 2.7.1]
 - name and version of MPI: [e.g. OpenMPI 4.1.1]

**Additional context**
Add any other context about the problem here.

--- a/include/openPMD/benchmark/mpi/MPIBenchmark.hpp
+++ b/include/openPMD/benchmark/mpi/MPIBenchmark.hpp
@@ -300,7 +300,7 @@ void MPIBenchmark<DatasetFillerProvider>::addConfiguration(
 template <typename DatasetFillerProvider>
 void MPIBenchmark<DatasetFillerProvider>::resetConfigurations()
 {
-    this->m_compressions.clear();
+    this->m_configurations.clear();
 }
 
 template <typename DatasetFillerProvider>

